### DISCUSSION TITLE
Gaia Resources queries at 29/11/2021

### DISCUSSION THREAD

**Discussion by GrantATgaia** [ternaustralia/ontology_tern#43] [Q&A] [answered]
Hi from the Gaia Resources team

A few questions ahead of tomorrow's TERN meeting. Hope this is an appropriate/useful forum to flag these?

- is it possible for TERN to provide a mapping of DwC to NDES? Even a partial mapping?
- we are keen to see the Excel TERN and python used in last week's presentation (if you are able to share it)
- some team members would like TERN to elaborate on how taxon is handled in the NDES
- is there a plan to expand the documentation and conceptual diagramming of the NDES with practical, descriptive examples?

Looking forward to the ongoing conversation and collaboration

Grant

**Comment by edmondchuc** [comment, marked as answer]
> is it possible for TERN to provide a mapping of DwC to NDES? Even a partial mapping?

@smguru - are you able to reply to this question?

> we are keen to see the Excel TERN and python used in last week's presentation (if you are able to share it)

The examples from last week are all in this repository https://github.com/ternaustralia/bdr-ibsa-sample-data. 

The [mapping spreadsheet](https://docs.google.com/spreadsheets/d/1Zon-3em4OLUoPwCm4KAK2I_VRmDsu8zt4Fm6zYsNGKc/edit?usp=sharing) we use in EcoPlots is also another example that may be useful.

> some team members would like TERN to elaborate on how taxon is handled in the NDES

Taxon information are captured with the `tern:Taxon` class as a result of an Observation.

> is there a plan to expand the documentation and conceptual diagramming of the NDES with practical, descriptive examples?

Yes, we are planning to create a cookbook of common modelling cases for ecological survey data.

**Reply by edmondchuc** [reply]
Edited comment above to add a link to the mapping spreadsheet used in EcoPlots.

**Comment by smguru** [comment]
1. yes we can provide a mapping example , please provide data of different DwC.
2. we can provide TERN mapping spreadsheet, 
3. Taxon can be an observable property or result. 
4. yes we will document the examples.

**Reply by GrantATgaia** [reply]
Thanks so much, Guru and Edmond.

Mieke will send through observation and specimen datasets for mapping.

Looking forward to the "cookbook".
